rotkehlchen/tests/unit/globaldb/test_upgrades.py

```diff
@@ -664,3 +730,273 @@ def test_applying_all_upgrade(globaldb: GlobalDBHandler):
     assert globaldb.get_setting_value('version', 0) == GLOBAL_DB_VERSION
     with globaldb.conn.cursor() as cursor:
         assert cursor.execute('SELECT COUNT(*) from assets WHERE identifier="eip155:/erc20:0x32c6fcC9bC912C4A30cd53D2E606461e44B77AF2"').fetchone()[0] == 1  # noqa: E501
+
+
+def test_update_consistency_with_packaged_db(
+        tmpdir_factory: 'pytest.TempdirFactory',
+        messages_aggregator: 'MessagesAggregator',
+):
```

Comment:
would be possible to add in each seaction of this test what you are checking? It is big and hard to follow and I bet that you will forget also. It is stated in the docstring but is hard to see what each block is for